specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-10-01-preview/Clusters.json

```diff
@@ -0,0 +1,567 @@
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ClusterPatch"
+            },
+            "description": "The parameters required to patch a Log Analytics cluster."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The existing cluster was successfully updated. Check provisioningStatus to see detailed status.",
+            "schema": {
+              "$ref": "#/definitions/Cluster"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ClusterErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ClusterProperties": {
+      "properties": {
+        "nextLink": {
+          "description": "The link used to get the next page of recommendations.",
+          "type": "string"
+        },
+        "clusterId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The ID associated with the cluster."
+        },
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The provisioning state of the cluster.",
+          "enum": [
+            "Creating",
+            "Succeeded",
+            "Failed",
+            "Canceled",
+            "Deleting",
+            "ProvisioningAccount",
+            "Updating"
+          ],
+          "x-ms-enum": {
+            "name": "ClusterEntityStatus",
+            "modelAsString": true
+          }
+        },
+        "isDoubleEncryptionEnabled": {
+          "type": "boolean",
+          "readOnly": false,
+          "description": "Configures whether cluster will use double encryption. This Property can not be modified after cluster creation. Default value is 'true'"
+        },
+        "isAvailabilityZonesEnabled": {
+          "type": "boolean",
+          "readOnly": false,
+          "description": "Sets whether the cluster will support availability zones. This can be set as true only in regions where Azure Data Explorer support Availability Zones. This Property can not be modified after cluster creation. Default value is 'true' if region supports Availability Zones."
+        },
+        "billingType": {
+          "type": "string",
+          "readOnly": false,
+          "description": "Configures whether billing will be only on the cluster or each workspace will be billed by its proportional use. This does not change the overall billing, only how it will be distributed. Default value is 'Cluster'",
+          "enum": [
```

Comment:
please add the "x-ms-enum" attribute with the appropriate settings

Reply:
Done